data/sql/updates/pending_db_world/rev_1612566366687490200.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612566366687490200');
+
+-- Relocate Strange Lockbox and add Bubbly Fissure near it.
+
+UPDATE `gameobject` SET `position_x` = 842.715, `position_y` = 2208.32, `position_z` = -136.765 WHERE `guid` = 27813;
+INSERT INTO `gameobject` (`id`, `position_x`, `position_y`, `position_z`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`) VALUES
+(177524, 838.26, 2208.14, -136.906, -0.753998, -0.656877, 900, 100, 1);
```

Comment:
You need to insert a GUID too. It is a Primary key of creature.
Nothing can be spawned without a spawnid

Reply:
And always delete before insert

Reply:
I can't delete an object that it never existed, this is a new gameobject for that bubbly. Also this table has auto-increment, so I don't think it is a good idea to create an specific id.

Reply:
We always delete before insert, codestyle of AC. Same as we always assign a GUID to a spawn.

Reply:
A codestyle can't be inferred by the code/PRs, it is a must to have this documented somewhere (there is nothing related in the wiki). It will help new contributors to just follow that and don't mess up while creating a PR (like me).

Reply:
Also for the GUID, there is no info about on how to generate a proper GUID. The one that is generated by Keira is not the one from the auto-increment and not even related to the last GUID.